From c54c2e6379e4f6b368ba5f00909185e2a4c5da92 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Tue, 20 Feb 2024 14:57:44 +0100
Subject: [PATCH 484/841] refactor(st): remove metadata compilation dependency

The file metadata.bin is independent from BL2, so it shouldn't be a
dependency of BL2 .stm32 file compilation. Remove it from STM32_DEPS.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git e54b75294d5bed29852a0f9a5afe14676c1f18fe refactor(st): remove metadata compilation dependency]
Change-Id: I47017a05678a4c5e09d7e0a90261110d520bcbad
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/361334
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 plat/st/common/common.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/plat/st/common/common.mk b/plat/st/common/common.mk
index 740fe244b..46118dfda 100644
--- a/plat/st/common/common.mk
+++ b/plat/st/common/common.mk
@@ -45,7 +45,6 @@ ifeq (${PSA_FWU_SUPPORT},1)
 NR_OF_FW_BANKS			:=	2
 NR_OF_IMAGES_IN_FW_BANK		:=	1
 JSON_METADATA			?=	plat/st/common/default_metadata.json
-STM32_DEPS			+=	${BUILD_PLAT}/metadata.bin
 
 $(eval $(call GEN_METADATA,${JSON_METADATA},${BUILD_PLAT}/metadata.bin))
 
-- 
2.47.2

